pkg/securitypolicy/framework.rego

```diff
@@ -342,6 +350,28 @@ exec_external := {"allowed": true} {
     workingDirectory_ok(process.working_dir)
 }
 
+default mount_encrypted_device := {"allowed": false}
+
+encrypted_device_mounted() {
+    data.metadata.encrypted_devices[input.target]
+}
+
+mount_encrypted_device := {"encrypted_devices": encrypted_devices, "allowed": true} {
+    not encrypted_device_mounted
+    encrypted_devices := {
+        "action": "add",
+        "key": input.target
```

Comment:
Missing a value. Metadata is a key/value store. What you probably want to do here is use a placeholder value (e.g. `input.target`) at which point I think you're fine.